### ISSUE TITLE
Fix stopped working with recent patches. 

### ISSUE THREAD

**Issue by SteinsM4te** [KingKrouch/AriseFix#3]
AriseFix is not working when playing recent version of Tales of Arise. 

After copying the files to Arise\Binaries\Win64 the game no longer fixes the Aspect Ratio. I've tried it on widescreen PC monitor & on the Steam Deck (16:10). 

It's very much possible that was caused by recent patches, as the latest release of AriseFix was September 2021. I know I'm digging in a graveyard, but there's no real alternative: Flawless Widescreen also stopped working (it can only fix FOV & HUD).

Would really appreciate the help/updating the fix.

**Comment by Naitrate** [comment]
Yeah, I'm aware, it broke a few game updates ago. I need to get around to fixing it, but my plans were to just make a singular All-In-One mod for Unreal Engine games, as most of them share the same issues.

I'm just way too burned out to work on a mod for another Unreal Engine game at the moment.

**Comment by SteinsM4te** [comment]
Understandable, have a nice day & take Your time!

**Comment by LoneStrider18** [comment]
Wait, does this mean there is still hope for this mod to be fixed? Also a new big DLC is coming in November to the game, so even if it got fixed before then, it would probably break again in a few months. Even so this mod was amazing for the game when it was working, and I am glad I got to enjoy it.

**Comment by Naitrate** [comment]
> Wait, does this mean there is still hope for this mod to be fixed? Also a new big DLC is coming in November to the game, so even if it got fixed before then, it would probably break again in a few months. Even so this mod was amazing for the game when it was working, and I am glad I got to enjoy it.

Maybe with the new update or DLC coming out, depends on how much longer Bandai Namco is constantly gonna update the game for. I just got really burnt out from modding Unreal Engine games or doing a lot of stuff manually in C++ with Assembly. Recovering from surgery right now, so I have plenty of time to kill anyways.

**Comment by LoneStrider18** [comment]
> Maybe with the new update or DLC coming out, depends on how much longer Bandai Namco is constantly gonna update the game for. I just got really burnt out from modding Unreal Engine games or doing a lot of stuff manually in C++ with Assembly. Recovering from surgery right now, so I have plenty of time to kill anyways.

Well, I hope whatever you got surgery for goes well, and you recover to full, good health. Like I said the mod was really cool, whether it comes back or not.

**Comment by LoneStrider18** [comment]
Any chance this will have a surprise "I updated it" Christmas drop? Jk, enjoy your Christmas, or whatever you choose to do this holiday, people.
